frontend/Gemfile

```diff
@@ -1,7 +1,7 @@
 eval(File.read(File.dirname(__FILE__) + '/../common_spree_dependencies.rb'))
 
-gem 'solidus_core', path: '../core'
-gem 'solidus_api', path: '../api'
 gem 'generator_spec', group: :test
+gem 'solidus_api', path: '../api'
+gem 'solidus_core', path: '../core'
```

Comment:
@Empact good point. @juliancheal would you mind disabling that cop in a separate commit and not reordering the Gemfile? 

Otherwise, I am fine with this as it does not change functionality and limits itself to style fixes.